src/System.Linq.Expressions/src/System/Dynamic/Utils/TypeExtensions.cs

```diff
@@ -117,19 +117,5 @@ public static MethodInfo GetAnyStaticMethod(this Type type, string name)
             }
             return null;
         }
-
-        public static MethodInfo[] GetMethodsIgnoreCase(this Type type, BindingFlags flags, string name)
-        {
-            var list = new List<MethodInfo>();
-            foreach (var method in type.GetRuntimeMethods())
-            {
-                // TODO: Binding flags filter
```

Comment:
This caused a regression compared the .NET Framework, changing the reported exception type in a test. See note and change in `ListInitExpressionTests.cs`.